app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ActiveRecord::Base
+	has_many :posts
+	has_many :comments
+	has_many :follows
+	acts_as_taggable
+
+	def to_s
+		first_name + " " + last_name
+	end
```

Comment:
this belongs in a `Decorator` (see the Draper gem) or view helper (unless you are using it for debugging or something)